### PR TITLE
Update GZBridge to be able to use gazebo airspeed. Add quadtailsitter.

### PR DESCRIPTION
**Pull request by jmackay2**
### Solved Problem
When adding a gz tailsitter, I found out that the current airspeed simulation is hard-coded to use the body velocity in the x direction. This doesn't work for the tailsitter, where the airspeed sensor needs to be rotated by 90 degrees to work correctly in fixed-wing flight. 

### Solution
This uncomments the gazebo airspeed integrations in the GZBridge and adds minor fixes to work correctly. This also adds a gazebo quadtailsitter configuration. 

### Changelog Entry
For release notes:
```
Feature: Adds the capability to use the Gazebo airspeed sensor and adds a Gazebo quadtailsitter.
```

### Test coverage
- Simulation of the standard_vtol model in gazebo.
- Simulation of the quadtailsitter. The quadtailsitter in fixed-wing is very fast and probably needs a little more modifications to be more realistic.

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d-posix/airframes/4004_gz_standard_vtol [hunk 1]
```diff
@@ -14,9 +14,7 @@ PX4_SIM_MODEL=${PX4_SIM_MODEL:=standard_vtol}
 param set-default SIM_GZ_EN 1
 
 param set-default SENS_EN_GPSSIM 1
-param set-default SENS_EN_BAROSIM 0
```
Comment (Jaeyoung-Lim) [inline ROMFS/px4fmu_common/init.d-posix/airframes/4004_gz_standard_vtol:17]: Why remove the barometer?

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]

**Review by Jaeyoung-Lim — APPROVED** [review]
LGTM!

**Comment by Jaeyoung-Lim** [comment]
@jmackay2 Could you rebase the PR so we can see if the SITL failures are real?

**Review by Jaeyoung-Lim — APPROVED** [review]
Thanks!

**Comment by daeyeonk** [comment]
Is this causing "make px4_sitl" to fail?

In file included from /root/PX4-Autopilot/src/modules/simulation/gz_bridge/GZBridge.cpp:34:
/root/PX4-Autopilot/src/modules/simulation/gz_bridge/GZBridge.hpp:105:47: error: 'AirSpeed' in namespace 'gz::msgs' does not name a type
  105 |         void airspeedCallback(const gz::msgs::AirSpeed &air_speed);
      |                                               ^~~~~~~~
compilation terminated due to -Wfatal-errors.
[1025/1038] Building CXX object src/modules/mavlink/CMakeFiles/modules__mavlink.dir/mavlink_messages.cpp.o
ninja: build stopped: subcommand failed.
make: *** [Makefile:232: px4_sitl] Error 1

**Comment by daeyeonk** [comment]
> > Is this causing "make px4_sitl" to fail?
> > In file included from /root/PX4-Autopilot/src/modules/simulation/gz_bridge/GZBridge.cpp:34: /root/PX4-Autopilot/src/modules/simulation/gz_bridge/GZBridge.hpp:105:47: error: 'AirSpeed' in namespace 'gz::msgs' does not name a type 105 | void airspeedCallback(const gz::msgs::AirSpeed &air_speed); | ^~~~~~~~ compilation terminated due to -Wfatal-errors. [1025/1038] Building CXX object src/modules/mavlink/CMakeFiles/modules__mavlink.dir/mavlink_messages.cpp.o ninja: build stopped: subcommand failed. make: *** [Makefile:232: px4_sitl] Error 1
> 
> It certainly sounds like it. What version of Gazebo are you using and what version of gz-sensors and gz-msgs do you have? Airspeed should have come into gz sensors and msgs about a year ago.
> 
> I see the air-speed library in gz-sensors in the latest gz versions: 7.3.0, and 8.2.0. I do not see it in gz-sensors 6 though.

I am using bash ./PX4-Autopilot/Tools/setup/ubuntu.sh for 22.04

**Comment by dagar** [comment]
We need to do something to keep builds working on 22.04.

**Comment by hamishwillee** [comment]
So this didn't go in? Worth still fixing the tiny bug:

image

**Comment by Jaeyoung-Lim** [comment]
We should probably switch to harmonic since that is the LTS. However, this would mean that we explicitly state that we are going to make a breaking change.

**Pull request merged**